packages/sw-runtime-caching/src/lib/constants.js

```diff
@@ -25,11 +25,18 @@
  * @type {string}
  * @memberof module:sw-runtime-caching
  */
-let cacheName = `sw-runtime-caching`;
- if (self && self.registration) {
+export const getDefaultCacheName = ({cacheId} = {}) => {
```

Comment:
Let's include some examples in the above comment of what the output from `getDefaultCacheName()` might look like. I got here from the constructor and still found myself wondering if a clear example was available.

From reading below it sounds like...

- If `cacheId` is specified (e.g `foo`, it's `foo-sw-runtime-caching`)
- If self/self.registration with scope `bar`, cacheName is `sw-runtime-caching-bar`. Does any naming related sanitization need to be done on scope here or do you think we should be fine as is?

Side: you explained why we're appending the scope in person and although this isn't the prettiest thing I understand why we're doing it the way we are.